src/CGT.jl

```diff
@@ -90,11 +89,11 @@ end
 
 function compute(sales::Sales{Currency{:EUR}}; details=false)
 	total_gain = by_period(sales; details)
-	net_gain = total_gain >= Allowance ? total_gain - Allowance : total_gain
-	tax = max(net_gain, 0EUR) * Tax
+	taxable_gain = max( total_gain - Allowance, 0EUR )
+	tax = taxable_gain * Tax
 
 	println("Total gain/loss: ", total_gain)
-	println("Net gain/loss excluding the allowance: ", net_gain)
+	println("Taxable gains (excluding the allowance): ", taxable_gain)
```

Comment:
I think the previous version is more accurate as the net can be a loss too.

Reply:
With net PnL adjusted to PnL - taxes, yes, I agree.